client/src/routes/[username]/+page.svelte

```diff
@@ -1,22 +1,65 @@
 <script lang="ts">
-    import User from '$lib/components/User.svelte';
-    import type { PageData } from './$types';
-    
-    export let data: PageData;
-
-    const { posts } = data;
+    export let data;
+    const { comics, user } = data;
 </script>
 
-
-<div class="user-container">
-    {#each posts || [] as post}
-        <User username={post.user.username}/>
-        <div>{post.title}</div>
-        <div>{post.content}</div>
-        <img src={`https://pub-57f47fc2081648efaa3f351802d15c5d.r2.dev/${post.image.path}`} alt="alo"/>
-    {/each}
+<div class="container">
+    <div class="sidebar">
+        <h1>Profile</h1>
+        <p><strong>Username: </strong>{user.username}</p>
+        <p><strong>Display Name: </strong>{user.displayname}</p>
+        <img src={user.profile_image.path} />
+    </div>
+    <div class="content">
+        <h1>Comics</h1>
+        {#each comics || [] as comic}
+            <a href={`/${user.username}/${comic.id}`}>
+                <div class="comic">
+                    <h2 class="comic-title">{comic.title}</h2>
+                    <p>{comic.description ?? "asdlfkjasdf"}</p>
```

Comment:
was gonna add an if statement around the element, but if the <p> is not there it might mess up with styling

```suggestion
                    <p>{comic.description ?? ""}</p>
```